src/java/fr/paris/lutece/plugins/blog/business/portlet/BlogPortlet.java

```diff
@@ -85,8 +86,9 @@ public BlogPortlet( )
     public String getHtmlContent( HttpServletRequest request )
     {
         GregorianCalendar calendar = new java.util.GregorianCalendar( );
-        Blog blog = BlogHome.findByPrimaryKey( this.getContentId( ) );
-        BlogPublication docPub = BlogPublicationHome.findDocPublicationByPimaryKey( this.getId( ), this.getContentId( ) );
+        Blog blog = BlogService.getInstance( ).loadBlog( this.getContentId( ) );
```

Comment:
We use the `BlogService.loadBlog()` method instead, to make sure we retrieve the documents / images associated with the blog